var/spack/repos/builtin/packages/py-bluepy/package.py

```diff
@@ -19,13 +19,13 @@ class PyBluepy(PythonPackage):
 
     depends_on('py-libsonata@0.1.7:0.999', type=('build', 'run'))
     depends_on('py-pandas@1.0.0:1.999', type=('build', 'run'))
-    depends_on('py-bluepy-configfile@0.1.15:0.999', type=('build', 'run'))
+    depends_on('py-bluepy-configfile@0.1.15:0.999,develop', type=('build', 'run'))
```

Comment:
@matz-e how dangerous is it to add ',develop' here?

I *think* it could allow a 'develop' to escape into a released version, is there a way to hint that it shouldn't be?

I'd like to have this, though, so that we can test bluepy w/ develop versions of its deps

Reply:
> I think it could allow a 'develop' to escape into a released version, is there a way to hint that it shouldn't be?

Normally the last numerical version is preferred. As this still bars `@<git sha>` commits, I'm not such a fan of it. In Spack, I'd rather remove the upper version restriction and add module tests to verify the deployment overall.